### PR TITLE
Getting_In_Touch.rst: Add COC reference link.

### PR DESCRIPTION
**Pull request by chhabrabhishek**
**For short term contributors:** we understand that getting your commits well
defined like we require is a hard task and takes some learning. If you
look to help without wanting to contribute long term there's no need
for you to learn this. Just drop us a message and we'll take care of brushing
up your stuff for merge!

### Checklist

- [x] I read the [commit guidelines](http://coala.io/commit) and I've followed
      them.
- [x] I ran coala over my code locally. (*All commits have to pass
      individually.* It is not sufficient to have "fixup commits" on your PR,
      our bot will still report the issues for the previous commit.) You will
      likely receive a lot of bot comments and build failures if coala does not
      pass on every single commit!

After you submit your pull request, **DO NOT click the 'Update Branch' button.**
When asked for a rebase, consult [coala.io/rebase](https://coala.io/rebase)
instead.

Please consider helping us by reviewing other peoples pull requests as well:

- pick up any PR at <https://coala.io/review>
- review it (check <https://coala.io/reviewing> for more info)
- if you are sure that it needs work, use `corobo mark wip <URL>` to get it out
  of the review queue.

The more you review, the more your score will grow at coala.io and we will
review your PRs faster!

### CODE REVIEW

**Comment by chhabrabhishek** [comment]
@anctartica is my commit incorrect syntactically or logically? I mean do I have a need to correct the content written in it or am I commiting it incorrectly?

**Comment by MacBox7** [comment]
Update `Added` to `Add`. You can refer [this](http://api.coala.io/en/latest/Developers/Writing_Good_Commits.html#shortlog) to understand why.

File: Help/Getting_In_Touch.rst [hunk 1]
```diff
@@ -24,3 +24,6 @@ Modularity, clean good code as well as a high usability for both users
 and developers of analyse routines (called bears) stand in the
 foreground of the development. We will not speed up our development if
 it needs sacrificing any aspect of quality.
+
+
+Please follow our Code Of Conduct, you can find it `here <http://docs.coala.io/en/latest/Help/FAQ.html#what-is-coala-community-code-of-conduct>`__.
```
Comment (jayvdb) [inline Help/Getting_In_Touch.rst]: This page is about contacting people.

This should be after "- Give us feedback", as a new item, 

> - Report any [Code of Conduct](link_to_coc) violations to ....

File: Help/.coafile [hunk 2]
```diff
@@ -0,0 +1,4 @@
+[cli]
+bears = SpaceConsistencyBear
+files = Getting_In_Touch.rst
+use_spaces = True
```
Comment (anctartica) [inline Help/.coafile]: don't need this. Why have you added a `.coafile` here? There is already one in the project that has the `SpaceConsistencyBear`

Comment (chhabrabhishek) [reply Help/.coafile]: @anctartica I did'nt added this file, I just ran the coala's space consistency bear on Getting_In_Touch.rst file and this .coafile generated automatically

Comment (anctartica) [reply Help/.coafile]: please remove it from the commit

File: Help/Getting_In_Touch.rst [hunk 3]
```diff
@@ -24,3 +24,10 @@ Modularity, clean good code as well as a high usability for both users
 and developers of analyse routines (called bears) stand in the
 foreground of the development. We will not speed up our development if
 it needs sacrificing any aspect of quality.
+
+
+Give Us Feedback
```
Comment (anctartica) [inline Help/Getting_In_Touch.rst]: Add the point below here: https://github.com/coala/documentation/pull/561/files#diff-15cbcde0631d4f2a91a45e04d0318b85R16

Comment (anctartica) [reply Help/Getting_In_Touch.rst]: don't need a new section

Comment (chhabrabhishek) [reply Help/Getting_In_Touch.rst]: but @anctartica can you see what @jayvdb had reviewed
he said me to add a new section

Comment (anctartica) [reply Help/Getting_In_Touch.rst]: he never mentioned a new section. 
> This should be after "- Give us feedback", as a new item

Comment (anctartica) [reply Help/Getting_In_Touch.rst]: where did he say to add a new section? @chhabrabhishek

Comment (chhabrabhishek) [reply Help/Getting_In_Touch.rst]: So what do I should change now?

Comment (anctartica) [reply Help/Getting_In_Touch.rst]: add a new point below the  "- Give us feedback" point

Comment (chhabrabhishek) [reply Help/Getting_In_Touch.rst]: and do i have to write report at community AT coala DOT io 
or do I simply write report at gitter and mail

Comment (anctartica) [reply Help/Getting_In_Touch.rst]: > If you believe someone is violating the code of conduct, we ask that you report it by emailing community AT coala DOT io.

Comment (anctartica) [reply Help/Getting_In_Touch.rst]: this is mentioned in COC

Comment (chhabrabhishek) [reply Help/Getting_In_Touch.rst]: Is it correct now? Or does it need any changes?

**Comment by ksdme** [comment]
Any opinion on if we should should have a wrapped link `-  If you believe someone is violating the code of conduct <http://docs.coala.io/en/latest/Help/FAQ.html#what-is-coala-community-code-of-conduct>__` to maintain line length?

**Comment by jayvdb** [comment]
ack https://github.com/coala/documentation/pull/561/commits/c596e8b39f6c07ab480b1c74c2140a7243d7fbc8

**Comment by jayvdb** [comment]
@gitmate-bot ff

**Comment by jayvdb** [comment]
While line length is important, this was a good attempt at balancing line length and readibility factors,  The result is mergable for user-docs.  I probably would have been stricter in dev-docs ;-)

**Pull request merged**